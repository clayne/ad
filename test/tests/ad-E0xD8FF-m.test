ad | -c always -E 0xD8FF -m | pjl-conductor-200.jpg | 0
