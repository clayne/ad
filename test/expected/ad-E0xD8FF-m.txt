[32m[K0000000000000000[m[K[36m[K:[m[K FF[41;1m[KD8[m[K [41;1m[KFF[m[KE1 020A 4578 6966 0000 4D4D 002A  .[41;1m[K..[m[K...Exif..MM.*
